### ISSUE TITLE
BingMapsGeocoderService crashes since Cesium 1.49

### ISSUE THREAD

**Issue by SunBlack** [AnalyticalGraphicsInc/cesium#7143]
Since Cesium 1.45 BingMapsGeocoderService is no longer the default geocoding service. In changelog following changes are recommend in case you want still use it

```
Cesium.BingMapsApi.defaultKey = 'yourBingKey';
var viewer = new Cesium.Viewer('cesiumContainer', {
    imageryProvider : new Cesium.BingMapsImageryProvider({
        url : 'https://dev.virtualearth.net'
    }),
    geocoder : [
            new Cesium.CartographicGeocoderService(),
            new Cesium.BingMapsGeocoderService()
    ]
});
```

This works fine until Cesium 1.48. But with 1.49 & 1.50 you got following error with this if you search for a city:

> An error occurred while rendering. Rendering has stopped.
RangeError: Invalid array length
RangeError: Invalid array length
    at E (http://localhost:8080/resources/Cesium/Cesium.js:524:27635)
    at S.createPotentiallyVisibleSet (http://localhost:8080/resources/Cesium/Cesium.js:524:30740)
    at Ze (http://localhost:8080/resources/Cesium/Cesium.js:525:21759)
    at Ye (http://localhost:8080/resources/Cesium/Cesium.js:525:19142)
    at at (http://localhost:8080/resources/Cesium/Cesium.js:525:28346)
    at st (http://localhost:8080/resources/Cesium/Cesium.js:525:28535)
    at we.render (http://localhost:8080/resources/Cesium/Cesium.js:526:6664)
    at A.render (http://localhost:8080/resources/Cesium/Cesium.js:532:16345)
    at t (http://localhost:8080/resources/Cesium/Cesium.js:532:9795)

**Comment by OmarShehata** [comment]
I can confirm this crash. Looks like you don't even need to change anything, it even happens in the Hello World.html. All you have to be doing is using the `Build/Cesium.js`. It doesn't happen in the Unminified version.

So far I've narrowed it to down `updateFrustums` in `View.js`. It seems that `near`, `far`, and `numFrustums` are passed in a `NaN`. I'm not sure why yet this would only be the case using the built version.

This also [came up on the forum](https://groups.google.com/d/msg/cesium-dev/y5tWLNuf--8/PcuJd-cPAgAJ).

**Comment by hpinkos** [comment]
@lilleyse @bagnell there seems to be an influx of camera related `NaN` crashes.  Did something change in the last two releases?

**Comment by hpinkos** [comment]
@mramato do you think this could be related to your geocoder fly to changes?

**Comment by mramato** [comment]
Yes, this has nothing directly to do with Bing, it's just getting triggered by their results.  I'll open a PR as soon as I get a chance.

**Comment by mramato** [comment]
Also added `next release` flag so I don't forget.

**Comment by cesium-concierge** [comment]
Congratulations on closing the issue! I found these Cesium forum links in the comments above:

https://groups.google.com/d/msg/cesium-dev/y5tWLNuf--8/PcuJd-cPAgAJ

If this issue affects any of these threads, please post a comment like the following:

> The issue at https://github.com/AnalyticalGraphicsInc/cesium/issues/7143 has just been closed and may resolve your issue. Look for the change in the next stable release of Cesium or get it now in the master branch on GitHub https://github.com/AnalyticalGraphicsInc/cesium.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by hpinkos** [comment]
Thanks for the bug report @SunBlack!  This has been fixed and will be included in the 1.51 release available November 1st.
